Jenkinsfile

```
@@ -44,4 +56,34 @@ pipeline {
       }
     }
   }
+  post {
```

Can we refactor the steps below?

```suggestion
post {
  when {
    GIT_BRANCH == 'latest'
   }
 }
```

Disclaimer: haven't tested this, but [should work in theory](https://jenkins.io/doc/book/pipeline/syntax/#when).

---

Alas no, sadly it says 
```
WorkflowScript: 60: Invalid condition "when" - valid conditions are [always, changed, fixed, regression, aborted, success, unstable, failure, notBuilt, cleanup] @ line 60, column 5.

       when {
```
Re: https://ci.news.tools.bbc.co.uk/blue/organizations/jenkins/psammead/detail/jenkins/8/pipeline

---

Shame! The only other thought is to refactor with closures:

```
def onLatestOnly(callback) {
  if (GIT_BRANCH == 'latest') {
    callback()
  }
}
```

Then lower down we'd:

```
onLatestOnly(notifySlack('danger', 'Aborted', gitCommitAuthor, stageName, gitCommitMessage, slackChannel))
```

---

Would be useful on line 43 too!